apis/core/lib/storage/s3.ts

```diff
@@ -1,5 +1,5 @@
 import aws from 'aws-sdk'
-import { isReadable } from 'isstream'
+import { readable } from 'is-stream'
```

Comment:
What is the difference between those two libraries?

Reply:
Not much I guess other than the new is only 2 years old while the other was last released 6 yrs ago :)